src/Client/HedgedConnections.h

```diff
@@ -188,6 +188,7 @@ class HedgedConnections : public IConnections
     Packet last_received_packet;
 
     Epoll epoll;
+    ContextPtr context;
     const Settings & settings;
```

Comment:
I'm not sure, but maybe just copy settings?

Reply:
I think it may be expensive - it's a big structure. Here we just copy ptr.

Reply:
Ok